src/main/java/world/bentobox/islandfly/listeners/FlyListener.java

```diff
@@ -43,9 +43,9 @@ public void onExitIsland(final IslandEvent.IslandExitEvent event) {
 
         final User user = User.getInstance(event.getPlayerUUID());
 
-        final Optional<Island> i = addon.getIslands().getIslandAt(user.getLocation());
+        final Island i = addon.getIslands().getIslandAt(user.getLocation()).orElse(null);
 
-        if (!i.isPresent()) return;
+        if (i == null) return;
```

Comment:
This code and the previous isPresent are exactly the same outcome.

Reply:
Yes, tho I wasn't able to call #isPresent anymore because the Island is not of Optional type anymore.
But I added #orElse which could give the null.